notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,1541 @@
+     ]
+    }
+   ],
+   "source": [
+    "hparams = prepare_hparams(yaml_file) ##\n",
+    "print(hparams)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create data loader\n",
+    "Designate a data iterator for the model. xDeepFM uses FFMTextIterator. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_creator = FFMTextIterator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create model\n",
+    "When both hyper-parameters and data iterator are ready, we can create a model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Add CIN part.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/anaconda/envs/py35/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.\n",
+      "  \"Converting sparse IndexedSlices to a dense Tensor of unknown shape. \"\n"
+     ]
+    }
+   ],
+   "source": [
+    "model = XDeepFMModel(hparams, input_creator)\n",
+    "\n",
+    "## sometimes we don't want to train a model from scratch\n",
+    "## then we can load a pre-trained model like this: \n",
+    "#model.load_model(r'your_model_path')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Train and fit model on validation data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "step 20 , total_loss: 3.7089, data_loss: 3.7089\n",
```

Comment:
Looks like after epoch 6 the loss metric starts to vibrate around some values (e.g., 1.2). There might be something to do with the hyperparameters. Since this notebook is just a quickstart, so it would be fine not to provide codes for tuning, but maybe it is worth showing the best parameters and results from experimentation that has tuning.